README.md

```diff
@@ -49,6 +49,23 @@ For a full list of Poetry cli options visit the official documentation [here](ht
 - `make dependency-tree` or `make d-tree` Displays a dependency tree.
 - `make fix-imports` Automatically format imports for all project python files.
 
+## Configuration
+
+To change Flask's default settings, supply the relevant environment variables with
+the FLASK_ prefix. For more details see [here](http://flask.pocoo.org/docs/1.0/config/#environment-and-debug-features)
```

Comment:
Better to avoid 'click here' links (ie link 'environment variables with the FLASK_ prefix' instead).

Reply:
updated